### PR TITLE
Fixing CEP-1422

### PR DESCRIPTION
**Pull request by janakiThevan**
Fixing CEP-1422

### CODE REVIEW

File: components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/endpoint/DataEndpointGroup.java [hunk 1]
```diff
@@ -319,11 +320,17 @@ public void run() {
...
+                        String[] urlElements = DataPublisherUtil.getProtocolHostPort(
+                                dataEndpoint.getDataEndpointConfiguration().getReceiverURL());
+                        if (!isServerExists(urlElements[1], Integer.parseInt(urlElements[2]))) {
+                            dataEndpoint.deactivate();
+                        }
+                    } catch (DataEndpointConfigurationException exception) {
+                        log.warn("Data Endpoint with receiver URL:" + dataEndpoint.getDataEndpointConfiguration().getReceiverURL()
+                                + " could not be deactivated", exception);
                     }
+
```
Comment (lasanthafdo) [inline components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/endpoint/DataEndpointGroup.java]: @janakiThevan 
Shall we remove the additional line break... ? No need to add line break unless there is a logical separation of code that needs to be highlighted..

Comment (janakiThevan) [inline components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/endpoint/DataEndpointGroup.java]: I'll remove that additional line..

Comment (janakiThevan) [inline components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/endpoint/DataEndpointGroup.java]: I removed the additional line break..

File: components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/DataPublisher.java [hunk 2]
```diff
@@ -143,7 +143,8 @@ private void processEndpoints(DataEndpointAgent dataEndpointAgent,
             boolean failOver = (Boolean) receiverGroup[0];
 
             DataEndpointGroup endpointGroup;
-            if (failOver) endpointGroup = new DataEndpointGroup(DataEndpointGroup.HAType.FAILOVER, dataEndpointAgent);
+            if (failOver)
```
Comment (suhothayan) [inline components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/DataPublisher.java]: add {} after if (failOver), based on WSO2 cording standard it has to be if (failOver) { ... }

Comment (janakiThevan) [inline components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/DataPublisher.java]: I'll fix the issue

**Pull request merged**